mon-pix/tests/unit/components/routes/campaigns/restricted/join-sco-test.js

```diff
@@ -559,7 +568,7 @@ describe('Unit | Component | routes/campaigns/restricted/join-sco', function() {
 
         // then
         sinon.assert.calledOnce(record.unloadRecord);
-        expect(component.errorMessage.string).to.equal('Vous êtes un élève ? <br/> Vérifiez vos informations (prénom, nom et date de naissance) ou contactez un enseignant.<br/><br/> Vous êtes un enseignant ? <br/> L‘accès à un parcours n‘est pas disponible pour le moment.');
+        expect(component.errorMessage.string.length).to.be.greaterThan(0);
```

Comment:
**question/suggestion:** Pourquoi ne pas invoquer `this.intl.t` comme pour les autres tests ?

**thought (non-blocking):** Tester la longueur du message ne correspond pas à l'intention du test qui est de vérifier l'affichage de `not found error`. D'ailleurs, ce test a probablement davantage sa place en intégration (rendering) https://guides.emberjs.com/release/testing/testing-components/#toc_testing-actions

**nitpick (if-minor):** Déplacer ce test en intégration sinon dans une PR de refacto, un jour

Reply:
Corrigé
Vu avec Mel : Les tests ont du sens en unitaire